pom.xml

```diff
@@ -56,39 +56,56 @@
     <sdk.antlr4.dir>${project.build.directory}/eforms-sdk/antlr4</sdk.antlr4.dir>
 
     <!-- Third-party libraries -->
-    <antlr4.version>4.9.3</antlr4.version>
-    <classindex.version>3.11</classindex.version>
-    <commons.lang3.version>3.12.0</commons.lang3.version>
-    <logback.version>1.2.11</logback.version>
-    <slf4j.version>1.7.36</slf4j.version>
+    <version.antlr4>4.9.3</version.antlr4>
+    <version.classindex>3.11</version.classindex>
+    <version.commons-lang3>3.12.0</version.commons-lang3>
+    <version.logback>1.2.11</version.logback>
+    <version.jackson>2.13.4</version.jackson>
+    <version.junit-jupiter-api>5.7.2</version.junit-jupiter-api>
+    <version.slf4j>1.7.36</version.slf4j>
 
     <!-- Plugins -->
-    <compiler.plugin.version>3.10.1</compiler.plugin.version>
-    <dependency.plugin.version>3.3.0</dependency.plugin.version>
-    <surefire.plugin.version>3.0.0-M7</surefire.plugin.version> <!-- Versions prior to 3.0.x do not pick up Junit 5 tests correctly. -->
+    <version.build-helper.plugin>3.3.0</version.build-helper.plugin>
+    <version.compiler.plugin>3.10.1</version.compiler.plugin>
+    <version.dependency.plugin>3.3.0</version.dependency.plugin>
+    <version.install.plugin>2.5.2</version.install.plugin>
+    <version.jacoco.plugin>0.8.8</version.jacoco.plugin>
+    <version.javadoc.plugin>3.4.0</version.javadoc.plugin>
+    <version.pgp.plugin>1.5</version.pgp.plugin>
+    <version.surefire.plugin>3.0.0-M7</version.surefire.plugin> <!-- Versions prior to 3.0.x do not pick up Junit 5 tests correctly. -->
   </properties>
 
   <dependencyManagement>
     <dependencies>
       <dependency>
         <groupId>org.atteo.classindex</groupId>
         <artifactId>classindex</artifactId>
-        <version>${classindex.version}</version>
+        <version>${version.classindex}</version>
       </dependency>
       <dependency>
         <groupId>org.slf4j</groupId>
         <artifactId>slf4j-api</artifactId>
-        <version>${slf4j.version}</version>
+        <version>${version.slf4j}</version>
       </dependency>
       <dependency>
         <groupId>ch.qos.logback</groupId>
         <artifactId>logback-classic</artifactId>
-        <version>${logback.version}</version>
+        <version>${version.logback}</version>
       </dependency>
       <dependency>
         <groupId>org.apache.commons</groupId>
         <artifactId>commons-lang3</artifactId>
-        <version>${commons.lang3.version}</version>
+        <version>${version.commons-lang3}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
+        <artifactId>jackson-databind</artifactId>
+        <version>2.13.1</version>
```

Comment:
If all version numbers are defined as properties, then it should also be the case for this one.
In addition, this does not seem to change the version for jackson-databind, it was already 2.13.1 before, as can be seen below

Reply:
Oops!
So, I did everything except for the real change :)